dev-docs/bidders/mobilefuse.md

```diff
@@ -1,21 +1,28 @@
 ---
 layout: bidder
 title: MobileFuse
-pbs: true
-pbjs: false
-media_types: banner, video, native
-tcfeu_supported: true
-gpp_supported: true
-schain_supported: true
+description: Prebid MobileFuse Bidder Adapter
+biddercode: mobilefuse
+tcfeu_supported: false
```

Comment:
Out of curiosity: you have a `gvl_id` , but do not support TCF in europe?

Reply:
Correct. We do support TCF but no longer operate in EU.

Reply:
That's interesting :bulb: